qlty-check/src/planner/config_files.rs

```diff
@@ -15,6 +16,27 @@ pub struct PluginConfigFile {
     pub contents: String,
 }
 
+#[derive(Debug, Clone, Serialize)]
+pub struct ConfigStagingOperation {
+    pub source_path: PathBuf,
+    pub destination_path: PathBuf,
+    pub operation_type: ConfigOperationType,
+}
+
+#[derive(Debug, Clone, Serialize)]
+pub enum ConfigOperationType {
+    /// Copy config from repository to staging area
+    CopyToStagingArea,
+    /// Copy config from repository to workspace root
+    CopyToWorkspaceRoot,
+    /// Load config from qlty directory to destination
+    LoadFromQltyDir,
+    /// Copy config into tool install directory
+    CopyToToolInstall,
+    /// Download and place fetched file
+    FetchFile,
```

Comment:
When the operation type is `FetchFile`, then where is the URL to fetch from stored in the plan?